cmd/cgoase/main.go

```diff
@@ -89,8 +97,7 @@ func main() {
 
 	conn, err := openDB()
 	if err != nil {
-		log.Printf("Failed to connect to database: %v", err)
-		return
+		return fmt.Errorf("Failed to connect to database: %v", err)
```

Comment:
Can we use the new error-handling with `%w`? (if go >= 1.13)

Reply:
Good point, thanks! I've updated all errors to use error wrapping.